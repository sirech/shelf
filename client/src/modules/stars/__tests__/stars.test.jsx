import React from 'react'
import renderer from 'react-test-renderer'

import Stars from '../stars'

describe('components', () => {
  describe('Stars', () => {
    it('renders correctly', () => {
      const stars = renderer.create(<Stars count={3} />)
      expect(stars.toJSON()).toMatchSnapshot()
    })
  })
})
