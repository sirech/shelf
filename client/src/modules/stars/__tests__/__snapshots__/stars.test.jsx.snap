exports[`components Stars renders correctly 1`] = `
<span
  className="undefined">
  <i
    className="fa fa-star"
    onClick={[Function]} />
  <i
    className="fa fa-star"
    onClick={[Function]} />
  <i
    className="fa fa-star"
    onClick={[Function]} />
  <i
    className="fa fa-star-o"
    onClick={[Function]} />
  <i
    className="fa fa-star-o"
    onClick={[Function]} />
</span>
`;
